app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
+  # def new
+  #   super
+  # end
```

Comment:
Normally, this controller will be used for overriding the functionality of devise controller (`ConfirmationsController` in this case) if it is unused, we can remove it kub